src/Simplify.elm

```diff
@@ -4045,6 +4051,83 @@ listTailChecks checkInfo =
                     []
 
 
+listMemberChecks : CheckInfo -> List (Error {})
+listMemberChecks checkInfo =
+    let
+        needleArg : Node Expression
+        needleArg =
+            checkInfo.firstArg
+
+        isNeedle : Node Expression -> Bool
+        isNeedle element =
+            Normalize.compare checkInfo element needleArg
+                == Normalize.ConfirmedEquality
```

Comment:
We should probably try to use `Normalize.compareWithoutNormalization`, in order not to normalize `needleArg` once for each element in the list. Instead we should try to do it only once (but also only when we need to).

Something like this could work I think:

```elm
isNeedle needle =
  let normalizedNeedle = Normalize.normalize needle
  in
  \element -> Normalize.compareWithoutNormalization (Normalize.normalize element) normalizedNeedle
```